sample-docker-app/Dockerfile.arm64v8

```diff
@@ -0,0 +1,12 @@
+FROM arm64v8/node:12.19.0-buster
```

Comment:
This image contains ARMv8 binaries, so how would this build process work on my x86_64 laptop?

Reply:
As I see this Dockerfile is only used for the arm64 version. The old one keeps as it is.

Reply:
If you use the docker xbuild and define the target platform, you can use the official node image

Reply:
Indeed that approach seems better